### ISSUE TITLE
DL3033 false positive when ARG is used to specify package version for yum install

### ISSUE THREAD

**Issue by lsoica** [hadolint/hadolint#1143]
For the following Dockerfile:

```
FROM rockylinux/rockylinux:9.6

ARG WHICH_VERSION=2.21-30.el9_6

RUN yum install -y which-2.21-30.el9_6 && yum clean all

RUN yum install -y which-${WHICH_VERSION} && yum clean all
```

hadolint reports:

```
$ hadolint Dockerfile 
Dockerfile:7 DL3033 warning: Specify version with `yum install -y <package>-<version>`.
```
